src/Sprites/ArtReader.cpp

```diff
@@ -13,6 +15,7 @@ namespace ArtReader {
 	void ReadAnimations(Stream::SeekableReader& seekableReader, ArtFile& artFile);
 	Animation ReadAnimation(Stream::SeekableReader& seekableReader);
 	Animation::Frame ReadFrame(Stream::SeekableReader& seekableReader);
+	void VerifyFrameCount(const ArtFile& artFile, std::size_t frameCount, std::size_t subframeCount, std::size_t unknownCount);
```

Comment:
What is this doing? The name is not so obvious.

Reply:
It is verifying the number of read frames, subframes (layers), and whatever the third optional count is counting equals the counts recorded in the .prt file. None of these values are needed for parsing the data, but since they are reported in the prt file, I figured they should be checked and ensure they match what was read.

I can add a comment explaining what it does or perhaps there is a better name?

Reply:
I think this is a case of using `Verify<NounPhrase>` rather than `Verify<PropertyOfNounPhrase>`. I think a better name would be better than a comment. A better name makes the code more self documenting, whereas a comment has to be kept in sync with the code.

Maybe `VerifyCountsMatchHeader`?

Reply:
Sounds good. I'll update.